build/configs/artik053/artik053_download.sh

```diff
@@ -64,15 +67,44 @@ main()
 				exit 1
 			fi
 
+			# Prepare for ROMFS
+			if [ "${CONFIG_FS_ROMFS}" == "y" ]; then
+				ROMFS_FLASHING_CFG=${OPENOCD_DIR_PATH}/romfs_flashing.cfg
+				if [ "${CONFIG_FS_ROMFS}" = "y" ]; then
```

Comment:
This if else condition is not required. The execution has reached this statement because ${CONFIG_FS_ROMFS}" == "y" is true as per above check. you can modify the script to avoid this unwanted check.
I can see that in both conditions, you are doing an echo to the file, so why it is initialized within condition. It should be unconditional

Reply:
I modified.